<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1174</width>
    <height>653</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1174</width>
    <height>653</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BSUIR-Express</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-1</y>
      <width>1191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Copperplate Gothic Light</family>
         <pointsize>25</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>BSUIR-Express</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 10px;</string>
    </property>
    <column>
     <property name="text">
      <string>Номер рейса</string>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время отъезда</string>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время прибытия</string>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип автобуса</string>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пункт назначения </string>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>1111</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Открыть файл с расписанием</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Показать все рейсы</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Показать автобусы в пути</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Показать автобусы в парке</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прибывающие 
до указанного времени</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Поиск по пункту назначения</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="back1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>110</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1174</width>
      <height>653</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>back1</zorder>
   <zorder>back2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>tableWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
